robot_skills/src/robot_skills/util/kdl_conversions.py

```diff
@@ -1,59 +1,245 @@
-# ROS
-import geometry_msgs.msg as gm
+import rospy
 import PyKDL as kdl
 
+# Needed for UGLY hacks
+import tf
+import time
 
-def point_msg_to_kdl_vector(point):
-    """ Converts a geometry_msgs.msg.Point to a kdl Vector
+import geometry_msgs.msg as gm
 
-    :param point: geometry_msgs.msg.Point
-    :return: kdl Vector
-    """
-    return kdl.Vector(point.x, point.y, point.z)
 
+class FrameStamped(object):
+    """Kdl alternative for a geometry_msgs.PoseStamped.
+    This class consists of a kdl.Frame and the frame_id w.r.t which the Frame is defined"""
 
-def quaternion_msg_to_kdl_rotation(quat):
-    """ Converts a geometry_msgs.msg.Quaternion to a kdl Rotation
+    def __init__(self, frame, frame_id, stamp=None):
+        assert isinstance(frame, kdl.Frame)
+        self.frame = frame
+        self.frame_id = frame_id
 
-    :param quat: geometry_msgs.msg.Quaternion
-    :return: kdl Rotation
-    """
-    return kdl.Rotation.Quaternion(quat.x, quat.y, quat.z, quat.w)
+    def __repr__(self):
```

Comment:
I used the first thing that worked. A better one-line representation is welcome. 
Preferable, some representation that can be copy-pasted into a console to construct the same value.